doc/compressed_transactions.md

```diff
@@ -0,0 +1,49 @@
+# Compressed Transaction Schema
+
+## 1. Abstraction
```

Comment:
In 61504d1a4d7ea545b1880686e426b1acae7d26f0:

`Abstraction` -> `Abstract`

Reply:
Fixed, Thanks!